api/WCharacter.h

```diff
@@ -22,6 +22,12 @@
 
 #include <ctype.h>
 
+// This Mentions that the following code is
```

Comment:
This is useless. It only changes how the function names are mangled. Since the functions are not compiled into an object file nor a static library, it doesn't matter if they're extern "C" or not. They're inline, in C++ and in C.

Reply:
I don't think this is true for any `inline` function, since they are not necessarily inlined. In this case, they are `always_inline` so it should indeed not matter, but it might be good to still have the `extern "C"` for good measure (and in case other non-always_inline functions are added later). I would omit the comment here, though, since the meaning/intention of `extern "C"` is rather obvious (and can be looked up if not) and I believe it's used without comment elsewhere too.